packages/filebrowser/src/browser.ts

```diff
@@ -138,6 +138,9 @@ export class FileBrowser extends Widget {
     this.layout.addWidget(this._crumbs);
     this.layout.addWidget(this._listing);
 
+    // We need to make the FileBrowser focusable so that it receives keyboard events
+    this.node.tabIndex = 0;
```

Comment:
I'd like to know if this is the right way to do this though. It's the only way I found for the filebrowser to receive keyboard events, otherwise, the body gets them (and keybindings don't get triggered).

Reply:
Ah, maybe that's was why I never managed to finish https://github.com/jupyterlab/jupyterlab/pull/6859

Reply:
I recall that @afshin dealt with some complex issues relating to focus and keybindings in the past. Perhaps he could weigh in?

Reply:
It looks this may actually address some of the accessibility issues mentioned https://github.com/jupyterlab/team-compass/issues/98#issuecomment-768585232.

Reply:
> I'd like to know if this is the right way to do this though. It's the only way I found for the filebrowser to receive keyboard events, otherwise, the body gets them (and keybindings don't get triggered).

I think you are right - I think only focusable elements can receive keyboard events (at least as targets), and setting the tabindex is how you make a div focusable. See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex for more.